src/scripting/ScriptContext.cpp

```diff
@@ -146,22 +146,29 @@ ScriptContext::ScriptContext(LuaSandbox& aLuaSandbox, const std::filesystem::pat
             return;
         }
 
-        m_vkBinds.emplace_back(acID, acDisplayName, wrapDescription(acDescription), wrapHandler(aCallback, acIsHotkey));
+        m_vkBinds.emplace_back(acID, acDisplayName, wrapDescription(acDescription), wrapHandler(aCallback, acIsHotkey), acIsOverlayHotkey);
```

Comment:
wrapHandler should be accepting the enum and make a proper wrapper, there should be another type added into the variant that inherits from normal hotkey without any special extensions

Reply:
looking at the definition of hotkeys, they are just std::function<type>...

probably for this to work correctly, you would want to change the usings into some actual structs holding an std::function first